naslib/defaults/trainer.py

```diff
@@ -8,12 +8,17 @@
 import torch
 import numpy as np
 
+import matplotlib.pyplot as plt
+import seaborn as sns
```

Comment:
1. Neither matplotlib.pyplot nor seaborn are used in this file. Remove.
2. Seaborn is missing in requirements.txt.